djstripe/models/billing.py

```diff
@@ -1295,8 +1301,8 @@ def is_status_temporarily_current(self):
 
         return (
             self.canceled_at
-            and self.start < self.canceled_at
             and self.cancel_at_period_end
+            and datetime.utcnow().replace(tzinfo=utc) < self.current_period_end
```

Comment:
This should probably be `timezone.now()` - currently I suspect this will fail if with `USE_TZ = False`.